### PR TITLE
add searchbar to mission browser

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- same as SP scenario browser

### CODE REVIEW

File: addons/ui/fnc_initDisplayRemoteMissions.sqf [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+params ["_display"];
+private _ctrlMaps = _display displayCtrl IDC_SERVER_ISLAND;
+private _ctrlMissions = _display displayCtrl IDC_SERVER_MISSION;
+
+ctrlPosition _ctrlMissions params ["_left", "_top", "_width", "_height"];
+
+private _ctrlSearch = _display ctrlCreate ["RscEdit", IDC_SEARCH];
+_ctrlSearch ctrlSetPosition [
+    _left,
```
Comment (mharis001) [inline addons/ui/fnc_initDisplayRemoteMissions.sqf]: `_left + 0.1 * GUI_GRID_W`

File: addons/ui/fnc_initDisplayRemoteMissions.sqf [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    _top,
+    _width - GUI_GRID_W,
```
Comment (mharis001) [inline addons/ui/fnc_initDisplayRemoteMissions.sqf]: `_width - 1.2 * GUI_GRID_W`

File: addons/ui/fnc_initDisplayRemoteMissions.sqf [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    GUI_GRID_H
```
Comment (mharis001) [inline addons/ui/fnc_initDisplayRemoteMissions.sqf]: `0.9 * GUI_GRID_H`

File: addons/ui/fnc_initDisplayRemoteMissions.sqf [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+_ctrlMissions ctrlSetPosition [
+    _left,
+    _top + GUI_GRID_H,
+    _width,
+    _height - GUI_GRID_H
+];
+_ctrlMissions ctrlCommit 0;
+
+// store all missions of the currently selected map
+private _fnc_storeMapMissions = {_this spawn {isNil { // delay a frame
```
Comment (dedmen) [inline addons/ui/fnc_initDisplayRemoteMissions.sqf]: why no execNextFrame? spawn could delay multiple frames.

Comment (commy2) [reply addons/ui/fnc_initDisplayRemoteMissions.sqf]: Doesn't work before postInit, and there is no postInit in i.e. 3den.

**Comment by commy2** [comment]
Kept the size of the edit box, because it looked a bit squashed, and moved the listbox below down a bit instead.

**Pull request merged**